python/tutorials/06-fused-attention.py

```diff
@@ -373,12 +581,14 @@ def bench_flash_attention(BATCH, H, N_CTX, D_HEAD, causal, mode, provider, dtype
     assert mode in ['fwd', 'bwd']
     warmup = 25
     rep = 100
+    causal = True
```

Comment:
Are we only testing `causal=True` in the benchmark?

Reply:
Yes. The bwd kernel is not yet updated to support `causal=False`. I will parameterize once that is done.

Reply:
Is this planned in this PR or in future PR?
If in future PR, can we move this later under the `if mode == 'bwd'` case so that we can still use causal=False when benchmarking the fwd kernel?